pkg/odo/cli/cli.go

```diff
@@ -31,7 +31,7 @@ const OdoRecommendedName = "odo"
 
 var (
 	odoLong = ktemplates.LongDesc(`
-Odo (OpenShift Do) is a CLI tool for running OpenShift applications in a fast and automated matter. Odo reduces the complexity of deployment by adding iterative development without the worry of deploying your source code.
+(OpenShift Do) odo is a CLI tool for running OpenShift applications in a fast and automated matter. odo reduces the complexity of deployment by adding iterative development without the worry of deploying your source code.
```

Comment:
Maybe `it reduces the ...`. The small letter at the start of the sentence looks weird.

Reply:
I know, but that's how we write `odo` now :( I'll add something else to make it look a bit better. Starting a sentence without capitalization looks odd.

Reply:
Maybe `it reduces...`?

Reply:
> I know, but that's how we write `odo` now :( I'll add something else to make it look a bit better. Starting a sentence without capitalization looks odd.

If you start the sentence you can use Odo ;-), or `odo` if it is as the code it looks better. 
If not in `code` it should be lowercase if not at the beginning of the sentence